tests/Functional/Schema/Db2SchemaManagerTest.php

```diff
@@ -30,4 +30,14 @@ public function testGetBooleanColumn(): void
         self::assertNull($columns['bool']->getComment());
         self::assertSame("That's a comment", $columns['bool_commented']->getComment());
     }
+
+    public function testSaveModeHandleOrphanForeignKeysBeforeIndexDrop(): void
+    {
+        $diff = $this->diffSaveModeHandleOrphanForeignKeysBeforeIndexDrop();
+
+        $this->assertSame([
+            'ALTER TABLE test_save_mode_orphan_fk_foreign DROP FOREIGN KEY FK_52D644CEA81E660E',
```

Comment:
These tests aren't really useful. Most likely you ran your code and copy/pasted the generated query as expected. The expected value is by no means a guarantee that the code will work as expected when it comes to interacting with a real database.